### PR TITLE
Fix #90 (removed most borders and shadows, reduced size of expand-arr…

### LINKED ISSUE #90: Within text, the component should be a tiny bit thinner
It would be nice if the component could blend better into scenarios like this:

https://kit-data-manager.github.io/webpage/typed-pid-maker/index.html

I suppose it would be possible to make it almost text-only. In this example it could be like this, without any borders, and therefore blend into text perfectly well:

v 21.11152/6ea60288-d895-414e-80c0-26c9fdd662b2 (copy)

, where v would be a nice arrow down, but not taller than the line-height.

The way it expands would need to be adjusted, but could look the same as of now. Maybe we can adjust all the padding/margin used currently to zero and animate it to the current values. Still, borders and other elements around may need to be made invisible or so in order to not interfere on the little space available.

### PR DESCRIPTION
**Pull request by ThomasJejkal**
…ow, removed 0-level copy button, all to fit into line height), changed scroll bars to auto-mode to avoid always rendering them, removed bg-white attributes for most elements to better harmonize with parent page, added optional include for local build in StoryBook to allow local testing w/o deployment

### CODE REVIEW

File: src/components/pid-component/pid-component.tsx [hunk 1]
```diff
@@ -257,7 +257,7 @@ export class PidComponent {
               <span
                 class={
                   this.currentLevelOfSubcomponents === 0
-                    ? 'group rounded-md shadow-md border text-clip inline-flex flex-grow py-0.5 px-1 open:align-top open:w-full ease-in-out transition-all duration-200 overflow-y-hidden font-bold font-mono cursor-pointer list-none bg-white overflow-x-hidden space-x-3 flex-nowrap flex-shrink-0 items-center'
+                    ? 'group text-clip inline-flex flex-grow py-0.5 px-1 open:align-top open:w-full ease-in-out transition-all duration-200 overflow-y-hidden font-bold font-mono cursor-pointer list-none overflow-x-hidden space-x-3 flex-nowrap flex-shrink-0 items-center'
```
Comment (maximiliani) [inline src/components/pid-component/pid-component.tsx:260]: Maybe some elevation from the background would be nice. It doesn't have to be a bright white background, but you could decrease the opacity of the background (e.g. bg-white/50) https://tailwindcss.com/docs/background-color. Additionally, I'd like some kind of rounded border around the component to make it more visible. If you're looking for a possibility to save space, you could decrease or inherit the font size.

**Review by maximiliani — APPROVED** [review]
Generally, I'd personally like to have a choice between the old, spacious and the new, compact style. The new design integrates much better into texts, but as a user I'd appreciate if the component was surrounded by a border to make it appear as one single unit/ element. But that's just my personal preference.

File: src/components/pid-component/pid-component.tsx [hunk 2]
```diff
@@ -300,26 +301,26 @@ export class PidComponent {
...
                     stroke-linecap="round"
                     stroke-linejoin="round"
                     stroke-width="1.5"
-                    viewBox="0 0 24 24"
-                    width="24"
+                    viewBox="0 0 12 12"
+                    width="12"
                   >
-                    <path d="M6 9l6 6 6-6"></path>
+                    <path d="M 2 2 l 4 8 l 4 -8"></path>
```
[the remark this answers is not in the document]
Comment (ThomasJejkal) [reply src/components/pid-component/pid-component.tsx:323]: Ok, I can try this out or maybe decrease the height of the arrow to have the same proportions as the old one (actually, I thought I did this already).

**Comment by maximiliani** [comment]
> > Generally, I'd personally like to have a choice between the old, spacious and the new, compact style. The new design integrates much better into texts, but as a user I'd appreciate if the component was surrounded by a border to make it appear as one single unit/ element. But that's just my personal preference.
> 
> I fully agree, but this will require more work. I'll have a look on it or you can try to make the modifications customizable, which is probably faster but I guess your agenda is quite packed.

Totally agree. If it's urgent, I recommend you looking into it. Otherwise, I can take a look after the 20th of December... There are also a lot of dependabot alerts...

**Pull request merged**